### DISCUSSION TITLE
Allure Report in Azure Pipeline

### DISCUSSION THREAD

**Discussion by tatiana-fox** [allure-framework/allure2#2477] [Questions & Support] [answered]
Hello team, I'm currently requiring assistance with implementing the Allure Report within our Azure pipeline for our Android App. Allure-results only consist of json files. We've installed 0, but are facing an issue which I've noted below: 'Job XXX_Android_UI_Test: Step  task reference is invalid. The task name PublishAllureReport is ambiguous. Specify one of the following identifiers to resolve the ambiguity: XXXTech.azure-pipelines-allure-report-publisher.PublishAllureReport.PublishAllureReport, qameta.allure-azure-pipelines.PublishAllureReport.PublishAllureReport'. 
We use this task:
 `- task: PublishAllureReport@1
  displayName: "publish allure report"
  inputs:
    testResultsDir: "path/to/allure-results"
    reportName: "Report 1"`

**Comment by tatiana-fox** [comment, marked as answer]
found solution, thanks

**Reply by PradeepBura** [reply]
What is the solution for this issue?
